mindsdb/integrations/handlers/ckan_handler/ckan_handler.py

```diff
@@ -1,7 +1,7 @@
 from ckanapi import RemoteCKAN as rc
 import pandas as pd
 
-from mindsdb.integrations.libs.base_handler import DatabaseHandler
+from mindsdb.integrations.libs.base import DatabaseHandler
```

Comment:
Normally we wouldn't do this. However, this PR is special because we wanted to merge it in time for Hacktoberfest (which, as you can see, we're delayed), which is why we included the renaming here.